src/annotator/components/Toolbar.js

```diff
@@ -1,36 +1,31 @@
-import { SvgIcon } from '@hypothesis/frontend-shared';
+import { IconButton } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
 
 /**
- * @param {object} props
- *  @param {import("preact").Ref<HTMLButtonElement>} [props.buttonRef]
- *  @param {boolean} [props.expanded]
- *  @param {string} [props.className]
- *  @param {string} props.label
- *  @param {string} props.icon
- *  @param {() => any} props.onClick
- *  @param {boolean} [props.selected]
+ * @typedef {import("@hypothesis/frontend-shared/lib/components/buttons").IconButtonProps} IconButtonProps
+ *
+ * @typedef {Omit<IconButtonProps, "className">} ToolbarButtonProps
  */
-function ToolbarButton({
-  buttonRef,
-  expanded,
-  className = 'Toolbar__button',
-  label,
-  icon,
-  onClick,
-  selected = false,
-}) {
+
+/**
+ * Style an IconButton for use on the Toolbar
+ *
+ * @param {ToolbarButtonProps} props
+ */
+function ToolbarButton({ ...buttonProps }) {
+  const { icon, title, ...restProps } = buttonProps;
   return (
-    <button
-      className={className}
-      aria-label={label}
-      aria-expanded={expanded}
-      aria-pressed={selected}
-      onClick={onClick}
-      ref={buttonRef}
-      title={label}
-    >
-      <SvgIcon name={icon} />
-    </button>
+    <IconButton
+      className={classnames(
+        'w-[30px] h-[30px]', // These buttons have precise dimensions
+        'flex items-center justify-center',
+        'border rounded bg-white text-grey-6 hover:text-grey-9 text-xl',
+        'shadow transition-colors'
+      )}
+      icon={icon}
+      title={title}
+      {...restProps}
+    />
```

Comment:
These annotator styles haven't, in many cases, been touched in several years, and have not been extracted as "patterns", etc. As such, expect to see some chattery utility styles in this round of changes as we convert things over. I'm hoping that meaningful patterns will surface for extraction and a second pass after main conversion will serve to pull 'em out as needed. For now I'm making use of local sub-components to encapsulate what might be future patterns or even shared components.